src/views/Dash.tsx

```diff
@@ -38,13 +39,35 @@ export default function Dash() {
     setProjectDetails(details);
   }
 
+  async function validityCheck(isValid: boolean) {
+    if (isValid) {
+      getProjectDetails();
+    } 
+    else {
+      try{
+        await window.contract.delete_project({ contract_address: pid });
+      }
+      catch (e)  {
+        Swal.fire({
+          title: "Error",
+          text: "There was an issue with deleting the project reference to the project that is DNE",
+          icon: "error",
+          confirmButtonText: "Ok",
+        });
+      }
+      navigate("/");
+      Swal.fire({
```

Comment:
Put the Swal inside of the `try` block. The code here will always be called regardless of if there is an error or not